src/server/cluster/cluster_family.cc

```diff
@@ -39,9 +39,9 @@ using namespace facade;
 using namespace util;
 
 using CI = CommandId;
-using ClusterShard = ClusterConfig::ClusterShard;
-using ClusterShards = ClusterConfig::ClusterShards;
-using Node = ClusterConfig::Node;
+using ClusterShardConfig = ClusterShardConfig;
+using ClusterShardConfigs = ClusterShardConfigs;
+using Node = ClusterNodeConfig;
```

Comment:
remove

Reply:
what do you mean by this comment? :thinking:

Reply:
don't forget to remove)